iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -394,7 +394,14 @@ public void tearDownTwin(DeviceState deviceState) throws IOException
         }
         if (internalClient != null)
         {
-            internalClient.closeNow();
+            try
+            {
+                internalClient.closeNow();
+            }
+            catch (IOException e)
+            {
+                // Ignore, the test isn't about being able to close the client
```

Comment:
why does it throw in those flaky instances? doesn't that indicate a problem? doesn't swallowing that exception mean we are ignoring a potential issue?

Reply:
We have tests where closing the client is part of the code under test, but this isn't one of them

Reply:
that doesn't explain why it throws here though

Reply:
I can take this out for now. I'm more interested in the TPM changes anyways

Reply:
I was under the impression that we generally don't fail a test if the cleanup fails, but this may be worth more discussion